.github/workflows/cclib_pytest.yml

```diff
@@ -25,10 +25,22 @@ jobs:
       - name: Install cclib
         run: |
           python -m pip install .
-      - name: Test with pytest
+      - name: Filter code
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            methods:
+              - 'cclib/cclib/method/**'
+      - name: Test core code with pytest
         run: |
           env | sort
           bash .github/scripts/run_pytest.bash
+      - name: Test method code with pytest
+        if: steps.filter.outputs.method == 'true' 
+        run: |
+          env | sort
```

Comment:
You can probably get rid of this `env | sort` since it will just print the same stuff as the one above.